### PR TITLE
First implementation

### PR DESCRIPTION
**Pull request by luvtechno**
Note: this repo is public. 

## What

Implemented the first working version of elevation plugin for Wantedly Design System. 

It comes with a very minimal UI.

image

### CODE REVIEW

File: src/my-command.js [hunk 1]
```diff
@@ -1,3 +1,65 @@
+const sketch = require('sketch')
```
Comment (luvtechno) [inline src/my-command.js:1]: https://github.com/BohemianCoding/SketchAPI

File: src/my-command.js [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+  const elevation = parseInt(outputString, 10)
+
+  const shadowY = calcShadowY(elevation)
+  const shadowBlur = calcShadowBlur(elevation)
+  const shadowSpread = calcShadowSpread(elevation)
+
+  document.selectedLayers.forEach(layer => {
+    layer.style.shadows = [
+      {
+        color: 'rgba(0, 0, 0, 0.02)',
```
Comment (luvtechno) [inline src/my-command.js:60]: https://developer.sketchapp.com/reference/api/#shadow

While the reference says this property is "A rgba hex-string (#000000ff is opaque black).", `rgba` style notation also works.

File: src/my-command.js [hunk 3]
```diff
@@ -1,3 +1,65 @@
...
+        x: '0',
+        y: '0',
```
Comment (luvtechno) [inline src/my-command.js:62]: Setting to `"0"` in `String` type. If this is set to numeric `0`, it defaults to `2`, which seems Sketch default value for shadow. 

Same for blur, it defaults to `4` if it is set to `0`...

File: src/my-command.js [hunk 4]
```diff
@@ -1,3 +1,65 @@
+const sketch = require('sketch')
+const UI = require('sketch/ui')
+
+function calcShadowY(elevation) {
+  if(elevation === 0) {
+    return '0'
+  }
+  return elevation / 2
```
Comment (KentoMoriwaki) [inline src/my-command.js]: How about always returning string type because It's difficult to expect what type will be returned?

**Comment by luvtechno** [comment]
Discussed with @yanAoym 

- Fix the shortcut. It currently conflicts with something else. `Command + Ctrl + S` maybe? 
- Rename the menu and repo name. It should be a name of our design system. For now, I'll put`"WtDesign"`  and `wantedly/sketch-wantedly-plugin`

File: src/my-command.js (hunk 4 above)
Comment (luvtechno) [reply src/my-command.js]: Ok, agreed. 

I like you are a typed person...!

**Review by KentoMoriwaki — APPROVED** [review]

**Pull request merged**